app/views/users/_self_introduction.html.erb

```diff
@@ -1,2 +1,2 @@
-<p><label for="user_email">Self Introduction</label><br/>
+<p><label class="label_account_request">Self-Introduction: (Please include a link to your website, etc.)</label><br/>
```

Comment:
It might be better to place "(Please include a link to your website, etc.)" as a hint, instead of making it a a part of the label

Reply:
Fixed